htdocs/web_portal/views/admin/added_service_type.php

```diff
@@ -4,7 +4,7 @@
 
     <?php require_once __DIR__ . '/../fragments/serviceTypeInfo.php'; ?>
 
-    <a href="index.php?Page_Type=Admin_Service_Type&amp;id=<?php echo $params['ID'] ?>">
+    <a href="index.php?Page_Type=Service_Type&amp;id=<?= $params['ID']; ?>">
```

Comment:
why isn't the php tag and echo needed here? I can see that it works, but I'm not at all sure why it works.

Reply:
Ahh. It was exceeding 80. So I have used another short version for  <?php echo i.e <?= ?> :)